CHANGELOG.md

```diff
@@ -465,6 +467,7 @@ Last release without a changelog :(
 [#1185]: https://github.com/linebender/druid/pull/1185
 [#1191]: https://github.com/linebender/druid/pull/1191
 [#1092]: https://github.com/linebender/druid/pull/1092
+[#1186]: https://github.com/linebender/druid/pull/1186
```

Comment:
The number 1186 should be changed to 1189 in both locations

Reply:
yup, fixed it, sorry for the amount of simple mistakes in the markdown.